tests/test_simple_unit.py

```diff
@@ -12,7 +12,9 @@
 from collections import namedtuple
 from datetime import datetime
 
-ppt = pp.pyparsing_test
+from pyparsing import testing
+
+ppt = testing
```

Comment:
we use the `pyparsing.testing` module instead of the `pyparsing.testing.pyparsing_test` class for namesapce.